pkg/framework/test/integration/integration_test.go

```diff
@@ -0,0 +1,53 @@
+package integration_test
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubectl/pkg/framework/test"
+)
+
+var _ = Describe("Integration", func() {
+	It("Successfully manages the fixtures lifecycle", func() {
```

Comment:
Would it make sense to have a benchmark test to see how long it takes to spawn and stop a fixture? I think that's a metric we'll be interested in.

Reply:
We introduced that in ["Track the time it takes to start&stop the fixtures"](https://github.com/totherme/kubectl/commit/a0e9df8b57e5c0bff3a4e4ef3f29f82dd0290cdc)

You can find the timings in our pipeline:
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/23#L5a1d829b:71
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/22#L5a1d90c7:27
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/20#L5a1d8f04:71
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/16#L5a1558c6:24
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/15#L5a1558c3:20
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/14#L5a15589a:19
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/13#L5a1558b8:62 (first test with parallel start)
- https://wings.concourse.ci/teams/k8s-c10s/pipelines/kubectl/jobs/test-dev-branch/builds/12#L5a1558b5:65 (last test with serial start)

Reply:
BTW: You can run our tests via `./scripts/run-tests.sh`; If you run `GINKGO_PERFORMANCE=1 ./scripts/run-tests.sh` it will also do some benchmarking.

Reply:
That's super exciting to see! The numbers are not that exciting. Let's see later what we can do.